azurepipelines/build/native/adu-ubuntu-amd64-build.yml

```diff
@@ -60,6 +60,17 @@ pr:
             - licenses/*
 
 jobs:
+    - job: BuildAduAgent_ubuntu2204
+      displayName: "Build ADU Agent - Ubuntu 22.04 (amd64)"
+      timeoutInMinutes: 60
```

Comment:
these are 60 in this file but the other files are 120. Should these be 120 too, or reason to keep at 60?